compose-builder/Makefile

```diff
@@ -256,6 +256,8 @@ gen:
 	$(GROVE_CHECK) \
 	DEV=$(DEV) \
 	APP_SVC_DEV=$(APP_SVC_DEV) \
+	CORE_EDGEX_REPOSITORY=$(CORE_EDGEX_REPOSITORY) \
+	CORE_EDGEX_VERSION=$(CORE_EDGEX_VERSION) \
```

Comment:
Also need `APP_SVC_REPOSITORY` and `APP_SERVICE_VERSION` put back

Reply:
good catch. forget about these

Reply:
actually there is no APP_SERVICE_VERSION to be overridden

Reply:
fixed